src/App.jsx

```diff
@@ -19,7 +19,7 @@ function App() {
      <Routes>
         <Route path='/' element={<Reviews reviews={reviews} setReviews={setReviews}/>} />
         <Route path="/reviews/:review_id" element={<SingleReview />} />
-        
+        <Route path="/reviews/:review_id/comments" element={<Comments/>} />
```

Comment:
You probably do not need this route Glen, this allows me go to a webpage (for example: /reviews/2/comments) and shows me a page of only comments with no review info.

Your singlereview component will sort or should be the one which renders the comments, rather than having a dedicated route.

Hope that makes sense!